maven-core/src/main/java/org/apache/maven/rtinfo/internal/DefaultRuntimeInformation.java

```diff
@@ -102,7 +102,9 @@ public boolean isMavenVersion(String versionRange) {
         Version current;
         try {
             String mavenVersion = getMavenVersion();
-            Validate.validState(StringUtils.isNotEmpty(mavenVersion), "Could not determine current Maven version");
+            if (!(mavenVersion != null && !mavenVersion.isEmpty())) {
```

Comment:
`getMavenVersion` is guaranteed to return a non null string (though it could be empty), so simplifying to `if (mavenVersion.isEmpty()) {`  looks a good idea.